test/db/anal/vars

```diff
@@ -151,8 +151,8 @@ afv
 EOF
 EXPECT=<<EOF
 arg int64_t arg3 @ r8
-arg int64_t arg2 @ rdx
```

Comment:
do you know how these arguments are sorted? and why the sorting has changed here?

Reply:
Yes, there are sorted by register index, which corresponds to the register order. x86-64 registers where sorted by merge sort, and where in a semi-random order. Now it matches the register order.

Reply:
Mmm could be, though it seems also that it tries to list things in the order of the ABI, could it be? (except for r8/r9 which seems to be wrong).

Reply:
That's also the register order. r8 is at offset 72, rcx 88, and rdx 96.